### ISSUE TITLE
Document steps for creating menu item/breadcrumbs for views

### ISSUE THREAD

**Issue by alwaysmashana** [NIHL/Drupal7-Migration#204]

**Comment by alwaysmashana** [comment]
http://nihlibrary.nihlibrary.com/admin/structure/views/view/tools/edit/default

I created this view to display on various pages (filtered by a particular taxonomy term). However, when I enter the URL path (e.g. services/bioinformatics-support/analysis-tools), the page will not appear in the secondary menu on the assigned services page. It only appears if the URL is shortened to the parent (e.g. services/bioinformatics-support/).

**Comment by gbohara** [comment]
Hi MaShana,
Are you trying to use the menu that already exists in the system as views path?

**Comment by alwaysmashana** [comment]
Yes. I’m adding the page to the menu. Isn’t that how you’re supposed to do it?

New page lives in Services, that URL path should be /services/[page name].

**Comment by svpatel1** [comment]
for MaShana, creating url paths & pages in teh view does not work for MaShana's account. Currently doing this as a workaround... need to revisit.
